### ISSUE TITLE
Javadoc errors when running under JDK 1.8

### ISSUE THREAD

**Issue by tobio** [campaignmonitor/createsend-java#25]
Java 8 introduces [doclint](http://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html) for Javadoc comments, much of the commentary in this library is incompatible with the linting rules. 

e.g

```
MavenReportException: Error while creating archive:Exit code: 1 - /Users/tobio/Documents/Projects/createsend-java/target/checkout/src/com/createsend/Administrators.java:28: error: bad use of '>'
[INFO]      * @throws CreateSendException Thrown when the API responds with HTTP Status >= 400
```

**Comment by wimdeblauwe** [comment]
If you don't want to fix everything, you can also turn doclint off: http://blog.joda.org/2014/02/turning-off-doclint-in-jdk-8-javadoc.html

**Comment by lukito** [comment]
@tobio this is now "fixed" in release v5.1.3..

**Comment by tobio** [comment]
:
